homeassistant/components/litterrobot/binary_sensor.py

```diff
@@ -0,0 +1,50 @@
+"""Support for Litter-Robot binary sensors."""
+from pylitterbot.enums import LitterBoxStatus
+
+from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import LitterRobotEntity
+from .hub import LitterRobotHub
+
+
+class LitterRobotBinarySensor(LitterRobotEntity, BinarySensorEntity):
```

Comment:
Why stuck this in between your base entity and the timing entity?
If you plan to possibly add more maybe you should go for entity descriptions right away?